src/remat.jl

```diff
@@ -66,19 +66,25 @@ end
 
 Base.eltype(R::ReMat) = eltype(R.z)
 
-function Base.copy!{S,R}(d::PooledDataVector{S,R}, s::PooledDataVector{S,R})
-    copy!(d.pool, s.pool)
+function Base.copy!(d::CategoricalVector, s::CategoricalVector)
```

Comment:
@nalimilan Thanks for the reminder.  I'll delete my version from the master branch of MixedModels.

It turns out that I may not actually call that method.  I need to do some more checking.